src/test/stepFunctions/commands/visualizeStateMachine.test.ts

```diff
@@ -303,43 +303,45 @@ describe('StepFunctions VisualizeStateMachine', async () => {
         assert.ok(managedVisualizations.get(mockTextDocumentTwo.uri.path))
     })
 
-    it('Test AslVisualizationManager managedVisualizations set removes visualization on visualization dispose, single vis', async () => {
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 0)
-
-        // Preview Doc1
-        mockVsCode.showTextDocument(mockTextDocumentOne)
-        let panel = await aslVisualizationManager.visualizeStateMachine(mockGlobalStorage)
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 1)
-
-        // Dispose of visualization panel
-        assert.ok(panel, 'Panel was not successfully generated')
-        panel = panel as vscode.WebviewPanel
-        panel.dispose()
-
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 0)
-    })
-
-    it('Test AslVisualizationManager managedVisualizations set removes correct visualization on visualization dispose, multiple vis', async () => {
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 0)
-
-        // Preview Doc1
-        mockVsCode.showTextDocument(mockTextDocumentOne)
-        let panel = await aslVisualizationManager.visualizeStateMachine(mockGlobalStorage)
-        console.log(panel)
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 1)
-
-        // Preview Doc2
-        mockVsCode.showTextDocument(mockTextDocumentTwo)
-        await aslVisualizationManager.visualizeStateMachine(mockGlobalStorage)
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 2)
-
-        // Dispose of first visualization panel
-        assert.ok(panel, 'Panel was not successfully generated')
-        panel = panel as vscode.WebviewPanel
-        panel.dispose()
-
-        assert.strictEqual(aslVisualizationManager.getManagedVisualizations().size, 1)
-    })
+    // TODO re-enable these tests once this race condition is resolved.
+    // https://github.com/aws/aws-toolkit-vscode/issues/1071
+    //
+    // it('Test AslVisualizationManager managedVisualizations set removes visualization on visualization dispose, single vis', async () => {
```

Comment:
why not `it.skip()` ?

Reply:
The doc says `* - _Only available when invoked via the mocha CLI._` which made me unsure if this works or not. Let me test

Reply:
Confirmed that it works, changed.